BuildResidentialScheduleFile/measure.rb

```diff
@@ -150,7 +150,7 @@ def create_schedules(runner, hpxml, epw_file, args)
 
     output_csv_path = args[:output_csv_path]
     unless (Pathname.new output_csv_path).absolute?
-      output_csv_path = File.expand_path(File.join(File.dirname(args[:hpxml_output_path]), output_csv_path))
+      output_csv_path = File.expand_path(output_csv_path)
```

Comment:
I noticed that this measure argument has some language ("Relative paths are relative to the HPXML output path") that now should be removed. Also check if any other argument descriptions need updating.